utils.py

```diff
@@ -85,7 +85,10 @@ def generate_key(num=20):
     return binascii.hexlify(os.urandom(num)).decode()
 
 def is_email(val):
-    return re.match("[^@]+@[^@]+\.[^@]+", val) # pylint: disable=anomalous-backslash-in-string
```

Comment:
nothing seems to be changed, then revert this, it is harder to review code when things like this are left in

Reply:
reverted